cookbooks/aws-parallelcluster-config/files/default/cloudwatch_agent/cloudwatch_agent_config_util.py

```diff
@@ -1,11 +1,11 @@
 """
-Validate and modify the data in the cloudwatch_log_files.json cookbook file.
+Validate and modify the data in the cloudwatch_agent_config.json cookbook file.
```

Comment:
I like the new name. Just out of curiosity, have any customers customized this file? It looks like for pcluster >= 3.0.0, modifying this file is undocumented (https://github.com/aws/aws-parallelcluster/wiki/CloudWatch-Logs) so we're probably good. @chenwany What do you think?

Reply:
I think there are some users customized logfile, TBH I don't know if they are using `cloudwatch_log_files.json`